q.cpp

```diff
@@ -0,0 +1,245 @@
+#include <stdio.h>
```

Comment:
I would move the function to where it should be. mongoBackend?
Then remove this file completely [ I have a comment about this already ].